pkg/config/templates/rancherd-10-harvester.yaml

```diff
@@ -76,6 +76,12 @@ resources:
     releaseName: harvester
     version: {{ .HarvesterChartVersion }}
     defaultNamespace: harvester-system
+    diff:
```

Comment:
The `diff` in `rancherd-13-monitoring.yaml` add the specific leaf objects,  here it seems skips all `status` of all `objects` under `harvester` managedchart, is the range too big ?

![image](https://user-images.githubusercontent.com/31133476/193034615-fbcb741c-c95b-4190-a991-224992836f68.png)

Reply:
yes, we should include a CRD name here. Good spot.

Reply:
Is the new fix verified?

I took a further look, and updated https://github.com/harvester/harvester/issues/2762.

It looks, the difference is, CRD `engineimages.longhorn.io` (and another 2) has generation `2` after system installation, but normal CRD has generation 1, that's why fleet-agent complain.

I did not find a clear clue, why the CRD (installed by helm) itself is updated right at the installation stage.  

If the fix makes `fleet-agent` happy, we can go ahead now.

![image](https://user-images.githubusercontent.com/31133476/193123441-7077bf53-8c12-4115-8d10-f7063f6cd4a8.png)

Reply:
yap, this is a new fix.
I thought that the root cause is like what you found. Also explain on [here](https://github.com/rancher/fleet/blob/master/docs/bundle-diffs.md)

> The bundled charts may have some objects that are amended at runtime.
> This leads the status of the bundle and associated GitRepo to be reported as "Modified"

>Fleet bundles support the ability to specify a custom [jsonPointer patch](http://jsonpatch.com/).
>With the patch, users can instruct fleet to ignore object modifications.

So that is what we want, instruct the fleet agent to ignore the modification at runtime.

I test with the running cluster and a new cluster. Looks like works well :D